src/main/java/io/seqera/tower/cli/Tower.java

```diff
@@ -67,8 +67,8 @@
     @Option(names = {"-u", "--url"}, description = "Tower server API endpoint URL. Defaults to tower.nf (TOWER_API_ENDPOINT)", defaultValue = "${TOWER_API_ENDPOINT:-https://api.tower.nf}", required = true)
     public String url;
 
-    @Option(names = {"-j", "--json"}, description = "Show output as JSON")
-    public boolean json;
+    @Option(names = {"--output"}, description = "Show output in defined format (only JSON is available so far)")
```

Comment:
Improve the description, it is not clear if the user has to pass `--output JSON`  or `--output json`.